sql/core/src/main/scala/org/apache/spark/sql/execution/ExpandExec.scala

```diff
@@ -42,9 +42,27 @@ case class ExpandExec(
   override lazy val metrics = Map(
     "numOutputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"))
 
-  // The GroupExpressions can output data with arbitrary partitioning, so set it
-  // as UNKNOWN partitioning
-  override def outputPartitioning: Partitioning = UnknownPartitioning(0)
+  /**
+   * The Expand is commonly introduced by the RewriteDistinctAggregates optimizer rule.
```

Comment:
What does this comment mean? I think we have the three cases for `Expand`: distinct aggregates, grouping analytics, and window aggregates.